src/components/chart/chart.tsx

```
@@ -212,37 +224,97 @@ export class Chart {
+                <div class="legend-content">
+                    {items.map((item) => {
+                        const isHidden = this.hiddenItems.has(item.text);
+                        return (
+                            <div
+                                class={{
+                                    'legend-item': true,
+                                    'legend-item--hidden': isHidden,
+                                }}
+                                key={item.text}
+                                onClick={() => this.handleLegendClick(item)}
+                            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)